src/components/Books.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Book from './sub-components/book';
+import AddNewBook from './sub-components/AddNewBook';
+
+export default function Books() {
+  const books = [
+    {
+      id: 1,
+      title: 'The Hunger Game',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: 2,
+      title: 'The Hunger Game',
+      author: 'Suzanne Collins',
+    },
+  ];
+  return (
+    <div>
+      {books.map((book) => (
+        <Book key={book.id} title={book.title} author={book.author} />
+      ))}
+      <AddNewBook />
+    </div>
+  );
```

Comment:
Renders a list of books using the `Book` component 🔥
Renders a form by using the CreateNewBook component 😎